spa/src/pages/Connected/index.tsx

```diff
@@ -57,74 +58,106 @@ const Connected = () => {
 		AP.navigator.go( "addonmodule", { moduleKey: "gh-addon-admin" });
 	};
 
-	return (<Wrapper>
-		<ConnectedContainer>
-			<div>
-				<HeaderImg src={colorMode === "dark" ? "/public/assets/jira-github-connected-dark-theme.svg" : "/public/assets/jira-github-connected.svg"} alt=""/>
-				<Title>GitHub is connected!</Title>
-				<TopContent>
-					It’s time to let everyone know GitHub’s ready to use and your<br />
-					team can use issue keys to link work to Jira.<br />
-				</TopContent>
-				<ButtonContainer>
+	return (
+		<Wrapper>
```

Comment:
Shouldn't be this be transformed to use `css`?

Reply:
This is React Comp Composition pattern 
Cant be transformed via emotion css props